app/src/main/java/erevacation/com/userlistanko/homescreen/AnkoHomeUI.kt

```diff
@@ -14,12 +14,13 @@ import org.jetbrains.anko.support.v4.viewPager
 
 class AnkoHomeUI : AnkoComponent<HomeActivity> {
 
-    var viewPager: ViewPager? = null
+    lateinit var viewPager: ViewPager
```

Comment:
Can this be private ?

Reply:
we cant, because then we cant access it from HomeActivity

Reply:
Ok